### PR TITLE
Refactor test scripts & enable E2E test on kubernetes/test-infra

### LINKED ISSUE #167: Integrate e2e test with kubernetes/test-infra
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like in detail**

**Describe alternatives you've considered**

**Additional context**

/assign

/kind test

### PR DESCRIPTION
**Pull request by chewong**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #167

**Special notes for your reviewer**:

**Release note**:
```

```

### CODE REVIEW

**Comment by chewong** [comment]
@andyzhangx could you add /ok-to-test to this PR? Thanks!

File: pkg/azuredisk/azuredisk.go [hunk 1]
```diff
@@ -26,20 +26,19 @@ import (
 	"time"
 	"unicode"
 
+	csicommon "sigs.k8s.io/azuredisk-csi-driver/pkg/csi-common"
```
Comment (andyzhangx) [inline pkg/azuredisk/azuredisk.go]: hi @chewong, are you going to do the same repo address rename for azurefile-csi-driver? We may make it consistent for these two drivers, thanks.

Comment (chewong) [reply pkg/azuredisk/azuredisk.go]: Opened https://github.com/kubernetes-sigs/azurefile-csi-driver/pull/127 to address this issue.

**Pull request merged**